phpcs.xml

```diff
@@ -15,4 +15,7 @@
     <file>src/</file>
     <file>tests/</file>
 
+    <!--Excluding config/bundles.php to avoid jenkins build failing-->
+    <exclude-pattern>config/bundles\.php</exclude-pattern>
```

Comment:
after every bundle install / update this files is regenerated by symfony and if a bundle name is long, code sniffer will complain about too long lines. I think we can have this exclusion to avoid some recurring headaches. 👍

Reply:
As Dani said - the main problem is that this file is re-generated after each install/update. And in case, each env would be specified in options for the bundle - line would easily exceed 120 chars. BUT, this is not any kind of a problem, because file is auto-generated. For that reason I decided to suppress this warning for that file